db/models.py

```diff
@@ -50,3 +55,68 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    row = models.IntegerField()
+    seat = models.IntegerField()
+    movie_session = models.ForeignKey(
+        to=MovieSession,
+        on_delete=models.CASCADE,
+        related_name="movie_session_tickets"
+    )
+    order = models.ForeignKey(
+        to=Order,
+        on_delete=models.CASCADE,
+        related_name="order_tickets"
+    )
```

Comment:
you overwhelm it too much
You have MovieSession object. For example

`my_session = MovieSession.objects.filter(show_date='2024-09-03', movie='Harry Potter').first()`
I know it's MovieSession, now I want to get all tickets of this movie
`my_session.movie_session_tickets.all() # it's a bit redundant to add "movie_session" in related_name because I know that my_session is MovieSession object`
so instead it's better to do
`my_session.tickets.all()`

Fix everywhere ;)